hironx_ros_bridge/rqt_plugin.xml

```diff
@@ -0,0 +1,22 @@
+<library path="src">
+  <class name="HiroNXDashboard" type="hironx_ros_bridge.hironx_dashboard.HiroNXDashboard" base_class_type="rqt_gui_py::Plugin">
+    <description>
+      A Python GUI plugin for displaying a dashboard that displays and interacts with low level hrpsys states.
+    </description>
+    <qtgui>
+      <group>
+        <label>Robot</label>
+        <icon type="theme">folder</icon>
+        <statustip>Plugins related to specific robots.</statustip>
+      </group>
+      <group>
```

Comment:
`rqt --force-discover` nor `--clear-config` didn't work for me.

Somehow `QtGui` couldn't find this via plugin mechanism. When I `catkin_make install`ed and source from its result (instead of sourcing from `devel`) I got it to work.